### DISCUSSION TITLE
Sentry-Native withouth crashpad_handler.exe

### DISCUSSION THREAD

**Discussion by Hrach97** [getsentry/support#32] [Support] [answered]
I use sentry-native. And I faced the problem of not being able to use crashpad_handler.exe. I can’t use .exe because I have to provide only .dll for the project.

Can I use sentry without crashpad_handler.exe?

What should I do if I can?

**Comment by bruno-garcia** [comment, marked as answer]
What type of app is it? Curious to know why it isn't possible to use the crash handler.

You have other options, depending on the platform you're targeting. For example, [Breakpad](https://docs.sentry.io/platforms/native/guides/breakpad/). And [In process](https://github.com/getsentry/sentry-native#compile-time-options)

> inproc: A small in-process handler which is supported on all platforms, and is used as default on Android.
